docs/en/overview/Architecture.md

```diff
@@ -298,11 +303,13 @@ close to `MUST_CACHE`, while still being able to persist the data. Since Alluxio
 <img src="{{ '/img/dataflow-async-through.gif' | relativize_url }}" alt="ASYNC_THROUGH data flow"/>
 </p>
 
+If you are writing with `ASYNC_THROUGH` and the worker crashes before you persist the data, then you will incur a data loss.
```

Comment:
depending on the data replications, if data is replicated to n copies and only n-1 workers crashed, there will be no data loss

Reply:
Yeah, only all copies lost will incur a data loss.